src/create.js

```diff
@@ -32,14 +32,15 @@ module.exports = {
  **
  */
 function newThingUI (createContext, dataBrowserContext, thePanes) {
+  if (!thePanes) throw new Error('@@ newThingUI: update API') // phase out
   const dom = createContext.dom
   const div = createContext.div
   if (createContext.me && !createContext.me.uri) {
     throw new Error('newThingUI:  Invalid userid: ' + createContext.me)
   }
 
-  var iconStyle = 'padding: 0.7em; width: 2em; height: 2em;' // was: 'padding: 1em; width: 3em; height: 3em;'
-  var star = div.appendChild(dom.createElement('img'))
+  const iconStyle = 'padding: 0.7em; width: 2em; height: 2em;' // was: 'padding: 1em; width: 3em; height: 3em;'
```

Comment:
Although I appreciate you improving the syntax in general, I generally try to avoid it for code that I don't touch as part of the purpose of the PR in general. All of these minor fixes makes the PR harder to review in general.

In general we might want to do some migration PRs,  where we only focus on improving the syntax without changing the logic. This makes it easier to review PRs, as they become more focused on specific themes of code change.